common/vm_remove_network_adapter.yml

```diff
@@ -14,8 +20,9 @@
     label: "{{ adapter_label | default(omit) }}"
     mac_address: "{{ netadapter_mac_addr | default(omit) }}"
     state: absent
-  register: remove_result
+  register: remove_adapter
+  ignore_errors: "{{ vm_remove_adapter_ignore_errors | default(False) }}"
```

Comment:
It seems Ansible usually use the lower case of "False"

Reply:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/bool_filter.html. 'False' or 'false' both work. But 'False' is the boolean value used by Python. Since Ansible is implemented by Python, I prefer to use False instead of false. We can discuss and unify the case after Diane returns.